src/hooks/kyberdao/index.tsx

```diff
@@ -518,6 +530,93 @@ export function useVotingInfo() {
   }
 }
 
+const aggregateValue = <T extends string>(
+  values: ({ [key in T]: string | number } | undefined)[],
+  field: T,
+): number => {
+  return values.reduce((acc, cur) => {
+    const value = cur?.[field] ?? 0
+    return (typeof value === 'number' ? value : parseFloat(value)) + acc
+  }, 0)
+}
+
+export function useGasRefundTier(): GasRefundTierInfo {
+  const { account, chainId } = useActiveWeb3React()
+  const kyberDaoInfo = useKyberDAOInfo()
+
+  const { data } = useSWR<GasRefundTierInfo>(
```

Comment:
stop using useSWR, axios, fetch.
Use rtk instead

Reply:
I'm just extending logic of this file. Someone might need to refactor them someday, but not me today.